state/upgrades.go

```diff
@@ -2190,6 +2190,33 @@ func AddModelLogsSize(pool *StatePool) error {
 	return nil
 }
 
+// AddModelLogsSize to controller config.
```

Comment:
This is only added to test the upgrade steps itself because right now there seems to be a bug that we expect it to be there

Reply:
This shouldn't be necessary. The construction of the controller config pushes the map through a schema validator that adds default values if they are not set.

Reply:
Resolving this as I took your patch